Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:12-alpine
+
+# Create app directory
+COPY . /usr/src/app
+WORKDIR /usr/src/app
+
+# Install app dependencies
+COPY package*.json ./
```

Comment:
Add the yarn lock file in this step as well to improve build caching

Reply:
Ah the repo doesn’t lock down the transitive dependencies.

Maybe @Bogdan-Lyashenko can explain his reasoning behind it